mcp-registry/servers/f1-mcp-server.json

```
@@ -0,0 +1,283 @@
+{
+    "name": "f1-mcp-server",
+    "display_name": "Formula One MCP Server",
+    "description": "A Model Context Protocol (MCP) server that provides Formula One racing data.",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Machine-To-Machine/f1-mcp-server.git"
+    },
+    "homepage": "https://github.com/Machine-To-Machine/f1-mcp-server",
+    "author": {
+        "name": "Machine To Machine"
+    },
+    "license": "MIT",
+    "categories": [
+        "Productivity",
+        "Knowledge Base",
+        "MCP Tools"
+    ],
+    "tags": [
+        "f1",
+        "formula one",
+        "racing",
+        "sports",
+        "data",
+        "mcp"
+    ],
+    "installations": {
+        "uvx": {
+            "type": "uvx",
+            "command": "uvx",
+            "args": [
+                "f1-mcp-server"
+            ],
+            "description": "Run the F1 MCP server globally using uvx (requires uv installed)",
+            "recommended": true
+        }
+    },
+    "arguments": {},
+    "is_official": true,
```

As explained in #128, this should be community
```suggestion
    "is_official": false,
```